client/apps/shipping-label/components/label-purchase-modal/packages-step/package-info.js

```diff
@@ -18,15 +18,15 @@ import Button from 'components/button';
 import getBoxDimensions from 'lib/utils/get-box-dimensions';
 import getFormErrors from '../../../state/selectors/errors';
 import {
-	updateWeight,
+	updatePackageWeight,
 	removePackage,
 	setPackageType,
 	openAddItem,
 } from '../../../state/actions';
 
 const renderPackageDimensions = ( dimensions, dimensionUnit ) => {
-	return `${ dimensions.length } ${ dimensionUnit } x 
-			${ dimensions.width } ${ dimensionUnit } x 
+	return `${ dimensions.length } ${ dimensionUnit } x
```

Comment:
Is there any case where this trailing space would help ensure the space after the `x`?